### DISCUSSION TITLE
addressing dust bias in next simulations

### DISCUSSION THREAD

**Discussion by mvertens** [NorESMhub/noresm3_dev_simulations#41] [Discussions about coupled simulations]
The CTSM PR [https://github.com/ESCOMP/ CTSM/pull/2803](https://github.com/ESCOMP/CTSM/pull/2803) should be relevant to what we want to do for our upcoming simulations to handle the dust biases.
This PR is NOT in CTSM master yet and also not in noresm2_5_alpha06 which I just made today.
However, it looks like independent of this PR, the main culprit for the dust bias is the setting
of the CTSM namelist z0param_method. It seems like changing
z0param_method = 'Meier2022'
two 
z0param_method="ZengWang2007"
resolved a lot of the dust bias - at least from my reading of the PR.

I proposing for upcoming simulation it would suffice to simply set 
z0param_method="ZengWang2007" in user_nl_clm? 

It looks like the changes for the PR actually involve code mods and should be namelist settings and we don't actually have a ctsm tag with these changes regardless.

@rosiealice @kjetilaas @oyvindseland agreed with this proposal.

**Comment by adagj** [comment]
Dust comparison between n1850.ne30_tn14.hybrid_fatessp.20240100 and NorESM2-MM piControl: https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/n1850.ne30_tn14.hybrid_fatessp.202401007_15_25_vs_N1850frc2_f09_tn14_20191001_1214_1224/website/html_img/plot_page_cb_DUST_ANN_LatLon_Mean.html

**Comment by rosiealice** [comment]
Hi @adagj!  Just wanting to note that I think the non-FATES run here is likely with the different BGC-predicted LAI, so that is perhaps the most likely reason for a bias. Would be iunteresting to see if the CLM-SP and FATES-SP give different dust fields. I would not expect them to be substantially different (famous last words...)
